harness/determined/keras/_tf_keras_context.py

```diff
@@ -40,6 +49,61 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self._wrapped_optimizers = []  # type: List[tf.keras.optimizers.Optimizer]
         self._compiled_optimizer = None  # type: Optional[tf.keras.optimizers.Optimizer]
 
+        # The following attributes may be configured via configure_fit().  Defaults match the
+        # normal keras.fit() defaults.
+        self._fit_verbose = True
+        self._fit_class_weight = None
+        self._fit_workers = 1
+        self._fit_use_multiprocessing = False
+        self._fit_max_queue_size = 10
+
+    def configure_fit(
+        self,
+        verbose: Optional[bool] = None,
+        class_weight: Any = _arg_not_provided,
+        workers: Optional[int] = None,
+        use_multiprocessing: Optional[bool] = None,
+        max_queue_size: Optional[bool] = None,
+    ) -> None:
+        """
+        Configure the following arguments to ``model.fit()``:
+
+           -  ``verbose``: defaults to ``True``. The Determined implementation of ``verbose=True``
```

Comment:
blocking: can we reformat this to be consistent with the other docstrings

Reply:
you mean swap `-` for `*` or what are you referring to?

Reply:
No reformat it to look like `wrap_dataset()`  (use `Args:`) if you build the docs in the current form, this docstring looks quite different from the other ones

Reply:
oh, you mean like the `Parameters` format I bet.  Actually, I'm thinking that we should link to the keras docs and call it a day here.  I don't see any reason for documenting anything, except maybe that the output of `verbose` is slightly different.

Reply:
aren't we exposing just a subset of the parameters listed in the keras docs?

Reply:
yes, but the subset we expose is clearly listed there without us duplication it in the docstring